packages/runtime/src/utils/branded.ts

```diff
@@ -1,5 +1,5 @@
 // FIXME: copy & paste from the builder repo, we need a `lib` package
 declare const __brand: unique symbol
-type Brand<B> = { [__brand]: B }
+export type Brand<B> = { [__brand]: B }
```

Comment:
Can't use a branded type in a reducer that is passed to the `combineReducers` call w/o having to export this.